components/ErrorLabel.js

```diff
@@ -6,7 +6,9 @@ import PropTypes from 'prop-types'
 export function ErrorLabel(props) {
   return (
     <div
-      className={`border-l-4 border-[#D3080C] mt-2 sm:-mb-6 px-10 py-2 bg-[#F3E9E8] font-body font-bold text-sm lg:text-p ${props.className}`}
+      className={`border-l-4 border-[#D3080C] mt-2 sm:-mb-6 px-10 py-2 md:mb-0 bg-[#F3E9E8] font-body font-bold text-sm lg:text-p ${props.className}`}
+      id="error1"
```

Comment:
It is possible to have error label on both create and join rooms.
Perhaps something more specific like joinRoomError, createRoomError

Reply:
fixed it